exla/lib/exla/defn.ex

```diff
@@ -233,7 +234,28 @@ defmodule EXLA.Defn do
   end
 
   @doc false
-  def __jit__(key, vars, fun, [args], options) do
+  def __jit__(key, vars, fun, args_list, options) do
+    {cache_funs, options} = pop_cache_funs(options)
+    __compile__(key, cache_funs, vars, fun, options).(args_list)
+  end
+
+  defp pop_cache_funs(options) do
+    case Keyword.pop(options, EXLA) do
+      {{_, _}, _} = popped ->
+        popped
```

Comment:
In which case we go this path?

Reply:
`EXLA.jit_cached?` but don't look at it. :D